.github/workflows/update_docker_versions.yaml

```diff
@@ -1,7 +1,9 @@
 ---
-name: Update Pulumi Docker image versions
+name: Update Docker image versions
 
 on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [develop]
```

Comment:
If we have weekly runs I wouldn't feel the need to also trigger these workflows on pushes.

(Didn't we recently remove that from some workflows :smile:?)

Reply:
Happy to do both/either. Dependabot was set up weekly, so I can copy that.

Reply:
Done in ec370e6da